Projects/MoveFlow/Sources/Service/DemoImplementation/MoveFlowDemoService.swift

```diff
@@ -0,0 +1,22 @@
+import hGraphQL
+
+public class MoveFlowDemoService: MoveFlowService {
+    public func sendMoveIntent() async throws -> MovingFlowModel {
+        return MovingFlowModel(
+            from: OctopusGraphQL.MoveIntentFragment(_dataDict: .init(data: [:], fulfilledFragments: .init()))
```

Comment:
We would like to avoid anything related to the octopus to be in here. 
Idea behind this services are to use octopus things only in the octopus implementation.

Can we initialise MovingFlowModel without graphql?